### DISCUSSION TITLE
Docker Compose Problem

### DISCUSSION THREAD

**Discussion by ChristianWi** [tomek199/RankMe#209] [Q&A]
Hi, I tried to build the current commit of the develop branch but i have trouble with docker compose. is docker compose working for you in the current branch - if not, what would be a commit, where it works?

**Comment by tomek199** [comment]
Hi, yes docker compose works for me in agains latest develop branch version. Here is my setup: 
- Java "17.0.1" 2021-10-19 LTS
- Gradle 7.5.1
- Docker 20.10.20
- Docker Compose v2.12.1. 
If you can share some stracktrace maybe I can help to figure out the issue on your local?

**Comment by dwi110** [comment]
hey im trying to test rankme but docker compose is giving me some problems for most services

 2024-06-11 14:16:09 exec /entrypoint.sh: no such file or directory 

any idea how to solve? running win10

**Reply by tomek199** [reply]
Hey, 

To be honest I haven't check it under Windows, but I guess it might be some problem with slash/backslash usage in relative paths in DOCKERFILE.
